src/telink/tools.mk

```
@@ -0,0 +1,235 @@
+# Telink Tools Download Makefile
+# This makefile provides targets to download Telink development tools
+
+# Configuration
+PROJECT_ROOT := ../..
+TOOLS_DIR := $(PROJECT_ROOT)/telink_tools
+DOWNLOAD_DIR := $(TOOLS_DIR)/downloads
+
+# Tool versions and URLs
+SDK_VERSION := 3.7.2.0
+SDK_REPO := https://github.com/telink-semi/telink_zigbee_sdk
+SDK_ARCHIVE := V$(SDK_VERSION).zip
+SDK_URL := $(SDK_REPO)/archive/refs/tags/$(SDK_ARCHIVE)
+
+# Telink toolchain URL
+TOOLCHAIN_URL := http://shyboy.oss-cn-shenzhen.aliyuncs.com/readonly/tc32_gcc_v2.0.tar.bz2
```

I can reimplement the verification logic later which I added in #162, but in the meantime could you please make it `https` instead of `http`?

```suggestion
TOOLCHAIN_URL := https://shyboy.oss-cn-shenzhen.aliyuncs.com/readonly/tc32_gcc_v2.0.tar.bz2
```

---

Fixed